netbench/netbench-scenarios/README.md

```diff
@@ -0,0 +1,144 @@
+# netbench-scenarios
+
+### Executable
+
+The executable includes a single default scenario: [`request_response`](https://github.com/aws/s2n-quic/blob/main/netbench/netbench-scenarios/src/request_response.rs). This sends `N` number of bytes to the server, which responds with `M` number of bytes. Several options are available for configuration:
+
+```shell
+$ cargo run --bin netbench-scenarios -- --help
+
+netbench scenarios
+
+USAGE:
+    netbench-scenarios [FLAGS] [OPTIONS] [OUT_DIR]
+
+FLAGS:
+    -h, --help
+            Prints help information
+
+        --request_response.parallel
+            Specifies if the requests should be performed in parallel
+
+    -V, --version
+            Prints version information
+
+
+OPTIONS:
+        --request_response.client_receive_rate <RATE>
+            The rate at which the client receives data [default: NONE]
+
+        --request_response.client_send_rate <RATE>
+            The rate at which the client sends data [default: NONE]
+
+        --request_response.count <COUNT>
+            The number of requests to make [default: 1]
+
+        --request_response.request_size <BYTES>
+            The size of the client's request to the server [default: 1KB]
+
+        --request_response.response_delay <TIME>
+            How long the server will take to respond to the request [default: 0s]
+
+        --request_response.response_size <BYTES>
+            The size of the server's response to the client [default: 10MB]
+
+        --request_response.server_receive_rate <RATE>
+            The rate at which the server receives data [default: NONE]
+
+        --request_response.server_send_rate <RATE>
+            The rate at which the server sends data [default: NONE]
```

Comment:
Its a little unclear how this rate should be formatted

Reply:
Yeah it's tricky without the default. Essentially it's your regular `42Mbps` or you can get fancy with `42MB/42ms`. I think I might add a section in the help that shows the format for each data type.

Reply:
added